pallets/rmrk-core/src/lib.rs

```diff
@@ -91,11 +91,9 @@ pub mod pallet {
 			+ codec::FullCodec;
 	}
 
-	/// Next available NFT ID.
 	#[pallet::storage]
 	#[pallet::getter(fn next_nft_id)]
-	pub type NextNftId<T: Config> =
-		StorageMap<_, Twox64Concat, T::CollectionId, T::NftId, ValueQuery>;
+	pub type NftIndex<T: Config> = StorageValue<_, T::NftId, ValueQuery>;
```

Comment:
Would this one need to be `NftId` instead of `T::NftId`?

Reply:
The concrete types are merged. Once this branch is merged/updated with the main branch, it will not be a problem

Reply:
merged main into this branch. resolved.